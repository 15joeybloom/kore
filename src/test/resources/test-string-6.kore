["\u00A9\u00a9"]

module TEST-STRING-6

endmodule []
