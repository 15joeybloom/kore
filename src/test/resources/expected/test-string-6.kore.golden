Definition
    { definitionAttributes =
        (Attributes 
            [ (MetaPattern (StringLiteralPattern (StringLiteral "\251")))
            ])
    , definitionModules =
        Module
            { moduleName = (ModuleName "TEST-STRING-6")
            , moduleSentences = []
            , moduleAttributes = Attributes []
            }
    }